requirements-testing.txt

```diff
@@ -1,4 +1,5 @@
 numpy>=1.21,<=1.23.5
+ipython>=7.34.0,<=8.12.0
```

Comment:
I am opening a separate PR for this here:

https://github.com/terrapower/armi/pull/1250

Fingers crossed!